docs/apm/service-maps.asciidoc

```diff
@@ -0,0 +1,50 @@
+[[service-maps]]
+=== Service maps
+
+beta::[]
+
+A service map is a real-time diagram of the interactions occurring in your application’s architecture.
```

Comment:
Not sure how we're dealing with capitalization of this feature, or features in general. The tab says "Service Map."

Reply:
We decided to lowercase both words to match other feature names. Conversation is here if you're interested: https://github.com/elastic/style-guide/issues/108